### ISSUE TITLE
How can I disable stacking estimator?

### ISSUE THREAD

**Issue by soliverc** [EpistasisLab/tpot#963]
All of my models produced by TPOT have some sort of stacking estimator, which is not ideal. I would just like something built in sklearn only, without having to install TPOT on other computers or environments. 

Is it possible to disable the stacking estimator?

**Comment by weixuanfu** [comment]
One workaround is to set a template for TPOT where Regressor/Classifior is only the last step of pipeline. (please see this [link](https://epistasislab.github.io/tpot/using/#template-option-in-tpot)). Then TPOT should not use stacking estimator.

**Comment by soliverc** [comment]
So in the docs it says to use the argument like:
```python
tpot_obj = TPOTClassifier(
                template='Selector-Transformer-Classifier'
                )
```
To remove the Stacking Estimator, I would simply set the template as classifier only like the below?:

```python
tpot_obj = TPOTClassifier(
                template='Classifier'
                )
```

**Comment by weixuanfu** [comment]
In both cases (template ='Selector-Transformer-Classifier' or template='Classifier'), StackingEstimator won't be used.
But if the template is something like 'Selector-Classifier-Transformer-Classifier', the StackingEstimator will be used for stacking the predictions from the 1st Classifier.

**Comment by soliverc** [comment]
Gotcha. Thanks.

**Comment by soliverc** [comment]
I can confirm that `template='Classifier'` just gives me a regular sklearn model.

**Comment by weixuanfu** [comment]
Yep, it should be. And `template ='Selector-Transformer-Classifier'` should give you a linear pipeline with 3 steps (1st: feature selection, 2nd: feature transformation; 3rd: a classifier) instead.

**Comment by soliverc** [comment]
Just a final question. I have already selected my features and processed them beforehand (imputing and scaling etc). So I don't really need `Selector-Transformer` steps, correct? Or would you still recommend it?

**Comment by weixuanfu** [comment]
If those preprocessing steps has done, they are not necessary in template. The main reason of adding those preprocessing steps into template is to let TPOT explore different combinations of feature selectors and transfomers for higher performance.
